components/webln/index.js

```diff
@@ -107,9 +115,8 @@ function RawWebLNProvider ({ children }) {
     })
   }, [setEnabledProviders])
 
-  const value = { provider: { ...provider, sendPayment: sendPaymentWithToast }, enabledProviders, setProvider }
   return (
-    <WebLNContext.Provider value={value}>
+    <WebLNContext.Provider value={{ provider: provider ? { sendPayment: sendPaymentWithToast } : null, enabledProviders, setProvider }}>
```

Comment:
I wanted a way for consumers of this context to not need to understand `provider` to use it ... so if it can be used, it's set. if it can't be used, it's null